src/app/store.ts

```diff
@@ -1,10 +1,15 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { todosSlice } from '../features/todos';
+import { filterSlice } from '../features/filter';
+import { currentTodoSlice } from '../features/currentTodo';
+import { TypedUseSelectorHook, useSelector } from 'react-redux';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices(todosSlice, filterSlice, currentTodoSlice);
```

Comment:
The 'combineSlices' function is not a valid function from '@reduxjs/toolkit'. You should use 'combineReducers' instead to combine the slices.